src/platform/packages/shared/kbn-test/src/functional_tests/start_servers/start_servers.ts

```diff
@@ -59,6 +59,38 @@ export async function startServers(log: ToolingLog, options: StartServerOptions)
           ],
     });
 
+    await runKibanaServer({
```

Comment:
Do I read this correct and this always starts the extra Kibana instance (same for `ruin_tests`)?
Asking because I think we would only want to do that for the tests that actually require this setup.
What do you think?

Reply:
I agree, just haven't found the best way to move the remote `runKibanaServer` somewhere else.